ddp-workspace/projects/ddp-sdk/src/lib/models/activity/createActivityInstanceResponse.ts

```diff
@@ -0,0 +1,9 @@
+import { BlockVisibility } from './blockVisibility';
+
+export interface CreateActivityInstanceResponse {
```

Comment:
my suggestion here would be:

interface NestedActivityInstance {
  // couple of words about this property
  blockVisibility?: BlockVisibility[];
}
export interface CreateActivityInstanceResponse extends ActivityInstanceGuid, NestedActivityInstance {}

public createInstance(): Observable<CreateActivityInstanceResponse | null> 
public deleteActivityInstance(): Observable<NestedActivityInstance | null> {

Reply:
Wouldn't we get a TSLint error `no-empty-interface`?

Reply:
we shouldn't. interface which extends multiple interfaces is new one